### DISCUSSION TITLE
Conditional card editor - show whether conditions fulfilled, always show card preview

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11241] [Other feature requests]
## The request

Here's how the Conditional card editor currently looks when the conditions do not match:

![image](https://i.imgur.com/ybFX7V3.png)

It completely hides the preview. This makes sense in a way, but it's not ideal imo. Here's an idea (mockup) that I think would make it a little more user-friendly.

![image](https://i.imgur.com/N6hVIc9.png)

Each individual condition would have an icon displayed in the top right corner. A green checkmark if that condition is currently fulfilled and a red X if not. This would allow the user to easily determine why their conditional card may not showing.

The other change is that the card preview would now be **shown at all times**.  Rather than hiding it and leaving a big empty space, tell the user that it's only being displayed in the card editor for preview purposes and won't be visible in their actual UI until all conditions are fulfilled. 

It could be refined further, rather than scary red error text like my mockup, maybe a short line with an icon or something:

| Icon | MDI icon              | Warning Text  |
|------|-----------------------|-------------|
| ![eye](https://i.imgur.com/r7e2psd.png)  | `mdi:eye-outline`     | All conditions are fulfilled, your card will be visible.     |
| ![hidden](https://i.imgur.com/wGt3If2.png)  | `mdi:eye-off-outline` | Card will not be visible since conditions not fulfilled. |

**Comment by zsarnett** [comment]
https://github.com/home-assistant/frontend/pull/5429

**Comment by bramkragten** [comment]
@zsarnett That PR doesn't fix it for the preview.

**Comment by zsarnett** [comment]
Oops. My mistake
